_inc/client/components/form/form-toggle/test/index.jsx

```diff
@@ -1,16 +1,20 @@
 /**
  * External dependencies
  */
-const assert = require( 'assert' ),
-	React = require( 'react' ),
-	ReactDOM = require( 'react-dom' ),
-	TestUtils = ReactDOM.TestUtils,
-	unique = require( 'lodash/uniq' );
+import assert from 'assert';
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import unique from 'lodash/uniq';
+/**
+ * External dependencies
+ */
+const TestUtils = ReactDOM.TestUtils;
 
 /**
  * Internal dependencies
  */
-const CompactFormToggle = require( 'components/forms/form-toggle/compact' );
+import CompactFormToggle from 'components/forms/form-toggle/compact';
 
 require( 'lib/react-test-env-setup' )();
```

Comment:
I can't figure out what this is referencing…

- Search indicates this is the only mention of it.
- Doesn't seem to point to any local file.

Reply:
We had a file like that in Calypso once. If it's not in JP, probably safe to remove.

Reply:
(Deferring to a follow-up PR since I don't wanna lose everyone's :heavy_check_mark:s)

Reply:
#11686